scss/_title-bars.scss

```diff
@@ -0,0 +1,51 @@
+.title-bar {
+  // scss-docs-start title-bar-css-vars
+  --#{$boosted-prefix}title-bar-padding-y: #{$title-bar-padding-y};
+  --#{$boosted-prefix}title-bar-font-size: #{$h2-font-size};
```

Comment:
Don't we rather need to create intermediate `$title-bar-*` Scss vars instead of using directly `$h2-font-size`, `$h2-spacing` and `$gray-500`?